GLAA.Web/FormLogic/IFieldConfiguration.cs

```diff
@@ -1,10 +1,11 @@
 using System.Collections.Generic;
-using GLAA.Web.Controllers;
 
 namespace GLAA.Web.FormLogic
 {
     public interface IFieldConfiguration
     {
         IDictionary<FormSection, FormPageDefinition[]> Fields { get; set; }
+
+        FormPageDefinition[] this[FormSection section] { get; }
```

Comment:
I've never seen this syntax before, what's it doing?

Reply:
I added this as shorthand to manipulate `Fields` a little easier. For future reference, this adds an index operator to the class - see here for more: https://stackoverflow.com/questions/424669/how-do-i-overload-the-operator-in-c-sharp